rules-tests/Php82/Rector/Class_/AllowDynamicPropertiesRector/Fixture/some_other_class.php.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rector\Tests\Php82\Rector\Class_\AllowDynamicPropertiesRector\Fixture;
+
+#[\AllowDynamicProperties]
+class AnotherObject
+{
+    public string $someProperty = 'hello world';
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php82\Rector\Class_\AllowDynamicPropertiesRector\Fixture;
+
+#[\AllowDynamicProperties]
+class AnotherObject
+{
+    public string $someProperty = 'hello world';
+}
+
+?>
```

Comment:
Second part is not needed if equal, also, please change fixture file name with `skip_` prefix, eg: `skip_already_has_allow_dynamic_properties.php.inc`:

```suggestion
```

Reply:
Fixed